### DISCUSSION TITLE
Any way to have upper/lower bounds not affect min/max labels?

### DISCUSSION THREAD

**Discussion by shogun168** [kalkih/mini-graph-card#815] [Q&A]
So right now, the two labels in the y-axis is supposed to show the lowest and highest data points for the time period. However, if you set a lower_bound lower than the lowest data point, it'll show that in the label instead. For graphs than involve %'s like humidity, these labels are now useless since it'll always say 0 and 100 (assuming that's the scale). It would still be good to show what the actual lows and highs are for the time period being shown in the graph imo. Not sure if this is a bug or by design ...

**Comment by ildar170975** [comment]
1) lower_bound is not set:
A bottom label corresponds to the minimal value (MIN) within a monitored time period.

2) lower_bound is set:
A bottom label is equal to lower_bound.
Sounds logical.






> For graphs than involve %'s like humidity, these labels are now useless since it'll always say 0 and 100 (assuming that's the scale)

Why useless? Labels are not for displaying min/max.
Use `show_extrema` feature instead if you need to know extrema values.

How do want to use `lower_bound` not affecting a bottom label?
Show a picture.

**Reply by shogun168** [reply]
Hmm, looks like I've misunderstood what the lower_bound and upper_bound labels on the graph were for. I've just tried the extrema feature and that's actually what I wanted (and assumed what the bounds were for). Thanks!
